src/middleware/changeURL.js

```diff
@@ -174,10 +172,7 @@ export const changeURLMiddleware = (store) => (next) => (action) => {
   if (pathname !== window.location.pathname || search !== window.location.search) {
     let newURLString = pathname;
     if (search) {newURLString += search;}
-    // if (pathname !== window.location.pathname) {console.log(pathname, window.location.pathname)}
-    // if (window.location.search !== search) {console.log(window.location.search, search)}
-    // console.log(`Action ${action.type} Changing URL from ${window.location.href} -> ${newURLString} (pushState: ${action.pushState})`);
-    if (action.pushState === true) {
+    if (action.pushState) {
       window.history.pushState({}, "", newURLString);
```

Comment:
> it doesn't appear that changing the browser window path has to go through Redux at all

Are you referring to this line?

Reply:
> but I'm not sure all what that function is doing

This is the way by which the URL updates when (e.g.) you change the color-by, or if you change dataset via the dropdowns, or how it used to change via the `_url` property. From comment at top of that file: 

What is this middleware?
This middleware acts to keep the app state and the URL query state in sync by intercepting actions and updating the URL accordingly.

Reply:
Good to know! I anticipated that middleware may be confusing (not many people use it, although plenty of libraries do underneath the surface). Could you amend the comment at the top of `changeURL.js` so that it's less confusing? Thanks